model/shared/src/main/scala/explore/model/ObsSummary.scala

```diff
@@ -121,6 +121,8 @@ case class ObsSummary(
   lazy val constraintsSummary: String =
     s"${constraints.imageQuality.label} ${constraints.cloudExtinction.label} ${constraints.skyBackground.label} ${constraints.waterVapor.label}"
 
+  def visualizationTimeOrNow: Instant = visualizationTime.getOrElse(Instant.now())
```

Comment:
Should this be a `F[Instant]`?

Reply:
In general we always have a viz time, in the obs is defined as optional but on the UI we try to get it from the db but if missing we default to `now`
That is done at a higher level so that models below assume there is one

Reply:
See
https://github.com/gemini-hlsw/explore/blob/master/explore/src/main/scala/explore/targeteditor/TargetTable.scala#L118

Reply:
Hmmm. Yeah, it probably should. I know we're doing this elsewhere, too. I'll see if I can find and fix those, too.

Reply:
I'm moving this to an extension method on Option[Instant]. I can see where it is possible to pass it as a "parameter", but sometimes we need it for every row in a table.

Reply:
This is no longer necessary at all.